olm/quickstarts/rhosak-openshift-connect-quickstart.yaml

```diff
@@ -3,160 +3,345 @@ kind: ConsoleQuickStart
+
+        1. Click the link to the Pod.
+           The **Pod details** page opens.
+
+        1. Click the **Terminal** tab.
+           A command-line window opens inside the Pod.
+
+        1. To check whether you can access the required tools, on the command line, enter `rhoas`.
+           The `rhoas` command should show help text for the RHOAS CLI.
 
       review:
         instructions: |-
-          #### Verify that you've successfully accessed the CLI tools:
-          Did you see the Red Hat OpenShift Application Services CLI help text in the terminal of the tooling pod?
+          #### Verify that you successfully installed the CLI tools
+           * Did you you see the RHOAS CLI help text on the command line of the Pod for the tools application?
         failedTaskHelp: This task isn’t verified yet. Try the task again.
+
       summary:
         success: >-
-          Great work! You have access to the tooling required for the remainder of this quickstart.
+            You've installed the required CLI tools. You're ready to verify connection between the RHOAS Operator and your OpenShift cluster.
         failed: Try the steps again.
 
-
-    - title: Connect your Red Hat OpenShift Application Services to your OpenShift instance.
+    - title: Installing the RHOAS Operator
       description: |-
-        With the tooling pod in place, you can connect your Red Hat OpenShift Streams for Apache Kafka instance to your OpenShift project/namespace.
+        The Red Hat OpenShift Application Services (RHOAS) Operator enables you to expose a Kafka instance in Streams for Apache Kafka to an OpenShift cluster.
 
-        The Red Hat OpenShift Application Services operator allows you to represent Red Hat Cloud Services as first class citizens in your OpenShift environment.
-        This enables you to work and integrate with these services using standard OpenShift/Kubernetes features and APIs.
+        In this task, you'll install the RHOAS Operator in your OpenShift cluster.
 
-        We will use the `rhoas` CLI to connect your Kafka instance to your OpenShift project. The `rhoas` CLI uses a browser-based login flow by default
-        (meaning, when you execute `rhoas login` it will start a sign-in flow in your web browser). Since our terminal in OpenShift does not have access to a browser,
-        we need to login using a token.
+        After you install the RHOAS Operator, you can use the RHOAS CLI to connect a specified Kafka instance to the cluster. You can then work directly with the Kafka instance using standard OpenShift features and APIs.
 
-        1. Navigate to [https://cloud.redhat.com/openshift/token](https://cloud.redhat.com/openshift/token) and copy the token to your clipboard using the **Copy to clipboard** button.
+        #### Prerequisites
+         - You can access your OpenShift cluster with the `dedicated-admin` role (OpenShift Dedicated) or `cluster-admin` role.
```

Comment:
@DuncanDoyle The Operator installation procedure was a carry-over from the service binding guide that we're delivering *outside* OpenShift. But you're right, of course - we won't need it here because the RHOAS Operator is already installed.

What we **will** need, however, is a procedure for installing the Service Binding Operator. I think it makes sense to explicitly install and use the SBO, instead of relying on (and having to explain) the fallback behaviour of the CLI, which is to use the SDK. I'll adapt this existing procedure for the Service Binding Operator instead.

Reply:
Yeah, makes sense.